boards/dptechnics/walter/doc/index.rst

```diff
@@ -0,0 +1,294 @@
+.. _walter:
+
+DPTechnics Walter
+#################
+
+Overview
+********
+Walter is a compact IoT development board that combines an Espressif ESP32-S3 SoC
+with a Sequans Monarch 2 GM02SP LTE-M/NB-IoT/GNSS modem.
+
+To help you get the most out of Walter, we have published the `schematic`_ and the
+`KiCAD footprint`_, along with our `datasheet`_ which describes all pins and test pads.
+
+
+.. figure:: img/board.jpg
+   :align: center
+   :alt: DPTechnic Walter board
+
+   DPTechnics Walter board (Credit: DPTechnics bv)
+
+Hardware
+********
+
+ESP32-S3-WROOM-1-N16R2 microcontroller:
+
+- Xtensa dual-core 32-bit LX7 CPU
+- 16 MiB quad SPI flash memory
+- 2 MiB quad SPI PSRAM
+- 150 Mbps 802.11 b/g/n Wi-Fi 4 with on-board PCB antenna
+- 2 Mbps Bluetooth 5 Low Energy with on-board PCB antenna
+
+Sequans Monarch 2 GM02SP modem:
+
+- Dual-mode LTE-M / NB-IoT (NB1, NB2)
+- 3GPP LTE release 14 (Upgradable up to release 17)
+- Ultra-low, deep-sleep mode in eDRX and PSM
+- Adaptive +23 dBm, +20 dBm and +14 dBm output power
+- Integrated LNA and SAW filter for GNSS reception
+- Assisted and non-assisted GNSS with GPS and Galileo constellations
+- Integrated SIM card
+- Nano-SIM card slot
+- u.FL RF connectors for GNSS and 5G antennas
+
+Inputs & outputs:
+
+- 24 GPIO pins for application use
+- UART, SPI, I²C, CAN, I²S, and SD available on any of the GPIO pins
+- ADC, DAC, and PWM integrated in ESP32-S3
+- 3.3 V software-controllable output
+- USB Type-C connector for flashing and debugging
+- 22 test points for production programming and testing
+- On-board reset button
+
+Power supply
+
+- 5.0 V via USB Type-C
+- 3.0 - 5.5 V via Vin pin
+- Not allowed to use both power inputs simultaneously
+- Designed for extremely low quiescent current
+
+Form factor
+
+- Easy to integrate via 2.54 mm headers
+- 55 mm x 24.8 mm board dimensions
+- Pin- and footprint-compatible with EOL Pycom GPy
```

Comment:
I suggest removing the extra `-`:
`Pin and footprint compatible with EOL Pycom GPy`